packages/apps/plugins/plugin-graph/src/types.ts

```diff
@@ -2,60 +2,20 @@
 // Copyright 2023 DXOS.org
 //
 
-import type { IconProps } from '@phosphor-icons/react';
-import type { getIndices } from '@tldraw/indices';
-import type { DeepSignal } from 'deepsignal';
-import type { FC } from 'react';
+import { Plugin } from '@dxos/react-surface';
 
-import type { Intent } from '@braneframe/plugin-intent';
-import type { Plugin } from '@dxos/react-surface';
-
-type Index = ReturnType<typeof getIndices>[number];
-
-export type MaybePromise<T> = T | Promise<T>;
-
-export type GraphNode<TData = any> = {
-  id: string;
-  index: Index;
-  label: string | [string, { ns: string; count?: number }];
-  description?: string;
-  icon?: FC;
-  data?: TData;
-  parent?: GraphNode;
-  onChildrenRearrange?: (child: GraphNode, nextIndex: Index) => void;
```

Comment:
does this become an action?

Reply:
no because it's not a user-facing thing, it's internal implementation details, it fell into `properties` along with the other excess stuff